web/html/html-structure/html-structure-define.md

```diff
@@ -28,52 +28,55 @@ The typical structure of HTML elements are defined as:
 
-
 ---
 ## Practice
 
 Identify the terms defining this HTML code:
 
-https://img.enkipro.com/26a6a342cc716551cb0fa5950d2da503.png
-
-A: ???
-B: ???
-C: ???
-D: ???
-E: ???
-F: ???
+```html
+<a href="https:/enki.com/">Enki</a>        
 
-* element
-* opening tag
-* end tag
-* attribute
-* attribute's name
-* attribute's value
-* content
-* value
-* rule
+element: ???
+opening tag: ???
+end tag: ???
+attribute: ???
+attribute's name: ???
+attributes value: ???
```

Comment:
Missing apostrophe? Should be "attribute's" no?

Reply:
Yes, I overlooked that grammar mistake